tests/test_team.py

```diff
@@ -0,0 +1,119 @@
+            for player in team.bench:
+                assert isinstance(player, Player)
+
+    @pytest.mark.vcr
+    def test_modifications(self, teams):
+        for team in teams:
+            assert isinstance(team.perm_attr, list)
+            for mod in team.perm_attr:
+                assert isinstance(mod, Modification)
+
+            assert isinstance(team.seas_attr, list)
+            for mod in team.seas_attr:
+                assert isinstance(mod, Modification)
+
+            assert isinstance(team.week_attr, list)
+            for mod in team.week_attr:
+                assert isinstance(mod, Modification)
+
+            assert isinstance(team.game_attr, list)
+            for mod in team.game_attr:
+                assert isinstance(mod, Modification)
+
+    def test_tarot(self, teams):
+        for team in teams:
+            assert isinstance(team.card, (Tarot, type(None)))
+
+    def test_misc(self, teams):
+        for team in teams:
+            assert isinstance(team.id, str)
+            assert isinstance(team.full_name, str)
+            assert isinstance(team.location, str)
+            assert isinstance(team.nickname, str)
+            assert isinstance(team.shorthand, str)
+            assert isinstance(team.slogan, str)
+            assert isinstance(team.emoji, str)
+            assert isinstance(team.main_color, str)
+            assert isinstance(team.secondary_color, str)
+
+            assert isinstance(team.shame_runs, (int, float))
+            assert isinstance(team.total_shames, int)
+            assert isinstance(team.total_shamings, int)
+            assert isinstance(team.season_shames, int)
+            assert isinstance(team.season_shamings, int)
+
+    def test_added_fields(self, teams):
+        for team in teams:
+            if getattr(team, "team_spirit", None):
+                assert isinstance(team.team_spirit, int)
+            if getattr(team, "rotation_slot", None):
+                assert isinstance(team.rotation_slot, int)
+            if getattr(team, "tournament_wins", None):
+                assert isinstance(team.tournament_wins, int)
+
+    @pytest.mark.vcr
+    def test_load(self):
+        bad_team = Team.load("3f8bbb15-61c0-4e3f-8e4a-907a5fb1565e")
+        assert isinstance(bad_team, Team)
+
+        with pytest.raises(ValueError):
+            bad_team = Team.load("00000000-0000-0000-0000-000000000000")
+
+    @pytest.mark.vcr
+    def test_load_by_name(self):
+        team = Team.load_by_name("Boston Flowers")
+        assert isinstance(team, Team)
+
+        bad_name = Team.load_by_name("Ohio Astronauts")
+        assert bad_name is None
+
+    @pytest.mark.vcr
+    def test_load_all(self):
+        teams = Team.load_all()
+        assert isinstance(teams, dict)
+        for key, team in teams.items():
+            assert isinstance(key, str)
+            assert isinstance(team, Team)
+            assert key == team.id
+
+    @pytest.mark.vcr
+    def test_load_at_time(self):
```

Comment:
make these separate test cases

I was about to make the same comment a bunch of times, so general guidance: if a test loads multiple objects, separate them into different tests that are explicitly checking for a specific output (ie in the simplest cases you have everywhere, one test for valid, one test for invalid)

like sure, unfortunately it's more boilerplate, but it's easier to debug in the future and will be better at pinpointing any issues that arise